docs/source/release.rst

```diff
@@ -73,11 +70,15 @@ follow SemVer, so typically a release will involve changing the version from
 a change in stable release versions from ``vX.(Y-1).Z`` to ``vX.Y.Z``,
 in the case of a MINOR version increase.
 
-----------------
+--------------
+Do the release
+--------------
+
+^^^^^^^^^^^^^^^^
 Create a new tag
-----------------
+^^^^^^^^^^^^^^^^
 
-Once the PR to ``release`` is approved, tag the new commit on release
+Once the PR to ``release`` is approved, tag the new commit on master
```

Comment:
If we do a PR from `master` to `release`, I think the new commit will be on `release`.
So which commit should be tagged on `master`? The last one before merging to `release`?

Reply:
I fixed this/ made it more clear, thanks!